### ISSUE TITLE
A11y_SQLAzureDataStudio_ConnectToASQLServer_ConnectionDetails_ScreenReader: Status message not announced by screen reader for the loading screen after hitting enter on 'connect' button.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9410]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yWCAG2.1;#A11yTCS;#SQLAzureDataStudio;#DesktopApp;#MacOs;#Benchmark;#A11y_SQLAzureDataStudioBenchmark;#A11ySev2;#MAS4.1.3;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Screen reader: Voice Over
MAS reference: MAS4.1.3

**Repro Steps:**
1.Launch Azure Data Studio.
2.Select the Connections icon (if not already selected) from left bar.
3.Hover over the 'Servers' header and select the 'New Connection' icon.
4.Navigate to connection details.
5.Launch Voice Over and navigate till connect button and press enter.
6.Check if proper status message is announced by screen reader.

**Actual:**
Status message about loading content is not announced when we press the connect button on connection details screen.

Same issue has been observed on following page:
1.Schema Compare->Menu Bar.(After selecting "compare" menu option from menu bar)

**Expected:**
Status message should be announced for the loading screen that is displayed after we hit connection button.

**User Impact:**
Screen reader users might not come to know what is the exact action being performed after they hit the 'connect' button.

**Attachment link reference:**
[9410_A11y_SQLAzureDataStudio_ConnectToASQLServer_ConnectionDetails_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4291066/9410_A11y_SQLAzureDataStudio_ConnectToASQLServer_ConnectionDetails_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
issue: announce the connecting/connected status information when making a connection.

**Comment by Charles-Gagnon** [comment]
What's the expectation here for connections being made? For this dialog is it enough to just announce "Connecting" when the spinner starts? And then rely on the OE tree to announce connected/expanded? or should the dialog announce "Connected" right before closing?

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_WCAG2.1;

**Comment by Raghu-addepalli** [comment]
https://user-images.githubusercontent.com/91586743/218711438-e168f1e3-a43a-48e6-8ee0-bf756cdc319f.mov

Verified the issue not it is not reproducing.

Git Hub Tags: #Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-SQLADS-MAC-Feb2020;
